lib/matplotlib/backend_bases.py

```diff
@@ -2559,6 +2674,120 @@ class NonGuiException(Exception):
     pass
 
 
+class WindowEvent(object):
+    def __init__(self, name, window):
+        self.name = name
+        self.window = window
+
+
+class WindowBase(cbook.EventEmitter):
+    """The base class to show a window on screen.
+
+    Parameters
+    ----------
+    title : str
+        The title of the window.
+    """
+
+    def __init__(self, title, **kwargs):
+        super(WindowBase, self).__init__(**kwargs)
```

Comment:
Yes, intentionally, when working with super, one should take of arguments as and when they get used.  I find this encourages good encapsulation of classes.